### ISSUE TITLE
totem of wratn NOT availabe, shaman lvl 66

### ISSUE THREAD

**Issue by BoogiemanNightmare** [dalaranwow/dalaran-wow#5574]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**:I am lvl 66 shaman, totem of wrath SHOULD be avail @ lvl 60, but I can not train it
**Expected behaviour**: totem of wrath should be avail @ lvl 60
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
No, it shouldn't. You don't learn Totem of Wrath (rank 1) from the trainers.
Closing

**Comment by BoogiemanNightmare** [comment]
So, please tell me where I go to get that? I searched forums and found nothing on it.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by Golganeth** [comment]
http://wowwiki.wikia.com/wiki/Totem_of_Wrath

**Comment by BoogiemanNightmare** [comment]
Thank you so VERY much :)

**Comment by Golganeth** [comment]
It's always a pleasure to help, but please next time make sure that you try to research a little bit more (asking players in-game, forums, internet reliable wotlk sources...) before opening a bug report!

Cheers and have a good one.

**Comment by BoogiemanNightmare** [comment]
I did, the posts were uninformative and no responses from other players

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________
